yt_dlp/extractor/radlive.py

```diff
@@ -0,0 +1,406 @@
+        self._sort_formats(formats)
+
+        data = info.get('structured_data', {})
+
+        release_date = data.get('releasedEvent', {}).get('startDate')
+        if release_date:
+            try:
+                release_date = datetime.strptime(release_date, '%Y-%m-%dT%H:%M:%S.%f%z').timestamp()
+            except ValueError:
+                release_date = None
+
+        channel = next(iter(_info.get('channels', [])), {})
+        channel_id = channel.get('lrn', '').split(':')[-1] or None
+
+        result = {
+            'id': video_id,
+            'title': info['title'],
+            'formats': formats,
+            'language': data.get('potentialAction', {}).get('target', {}).get('inLanguage'),
+            'thumbnail': data.get('image', {}).get('contentUrl'),
+            'description': data.get('description'),
+            'release_timestamp': release_date,
+            'channel': channel.get('name'),
+            'channel_id': channel_id,
+            'channel_url': f'https://rad.live/content/channel/{channel_id}' if channel_id else None,
+
+        }
+        if content_type == 'episode':
+            result.update({
+                # TODO: Get season number when downloading single episode
+                'episode': info.get('title'),
+                'episode_number': info.get('number'),
+                'episode_id': info.get('id'),
+            })
+
+        return result
+
+
+class RadLiveSeasonIE(RadLiveIE):
+    IE_NAME = 'radlive:season'
+    _VALID_URL = r'https?://(?:www\.)?rad\.live/content/season/(?P<id>[a-f0-9-]+)'
+    _TESTS = [{
+        'url': 'https://rad.live/content/season/08a290f7-c9ef-4e22-9105-c255995a2e75',
+        'md5': '40b2175f347592125d93e9a344080125',
+        'info_dict': {
+            'id': '08a290f7-c9ef-4e22-9105-c255995a2e75',
+            'title': 'Bad Jokes - Season 1',
+        },
+        'playlist_mincount': 5,
+    }]
+
+    @classmethod
+    def suitable(cls, url):
+        return False if RadLiveIE.suitable(url) else super(RadLiveSeasonIE, cls).suitable(url)
+
+    def _real_extract(self, url):
+        season_id = self._match_id(url)
+        webpage = self._download_webpage(url, season_id)
+
+        _info = json.loads(self._search_regex(
+            r'<script[^>]*type=([\'"])application/json\1[^>]*>(?P<json>{.+?})</script>',
+            webpage, 'video info', group='json'))['props']['pageProps']['initialContentData']
+        info = _info['season']
+
+        entries = []
+        for episode in info['episodes']:
+            entries.append({
+                '_type': 'url_transparent',
+                'url': episode['structured_data']['url'],
+                'series': next(iter(_info.get('series', [])), {}).get('title'),
+                'season': info['title'],
+                'season_number': info.get('number'),
+                'season_id': info.get('id'),
+                'ie_key': RadLiveIE.ie_key(),
+            })
```

Comment:
```suggestion
        entries = [{
            ...
        } for episode in info['episodes']]
```

Also, it is good if you can also return `id` in the infodict (if you can get accurate id)